geo.go

```diff
@@ -21,3 +21,84 @@ func Haversine(startLat, startLon, endLat, endLon float64) float64 {
 	d := EarthRadiusKm * c
 	return d
 }
+
+// Rounding function courtesy David Vaini
+// https://gist.github.com/DavidVaini/10308388
```

Comment:
how is this different from the [stdlib math.Round](https://golang.org/pkg/math/#Round)?

Reply:
It is to ensure would work with earlier versions of Go--apparently Round is new.